### PR TITLE
{lang}[foss/2019b and fosscuda/2019b] R v3.6.2

### PR DESCRIPTION
**Pull request by edmondac**
(created using `eb --new-pr`)

### CODE REVIEW

**Comment by edmondac** [comment]
This is taken from R 3.6.1 from https://github.com/easybuilders/easybuild-easyconfigs/pull/9365. Then I bumped it to 3.6.2, ran easy_update and injected checksums.

Building now on my test machine

**Comment by edmondac** [comment]
Incorporates https://github.com/easybuilders/easybuild-easyconfigs/pull/9514 - so there will be two fewer eb files in here once that's merged

**Comment by boegel** [comment]
@edmondac Can you sync this with current develop now #9504 (GSL), #9507 (UDUNITS), #9514 (cairo + pixman) are merged?

`eb --sync-pr-with-develop 9519` (assuming you're using EasyBuild v4.1.0)

File: easybuild/easyconfigs/g/GSL/GSL-2.6-GCCcore-8.3.0.eb [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+
+name = 'GSL'
+version = '2.6'
+
+homepage = 'https://www.gnu.org/software/gsl/'
+description = """The GNU Scientific Library (GSL) is a numerical library for C and C++ programmers.
+ The library provides a wide range of mathematical routines such as random number generators, special functions
+ and least-squares fitting."""
+
+toolchain = {'name': 'GCCcore', 'version': '8.3.0'}
```
Comment (boegel) [inline easybuild/easyconfigs/g/GSL/GSL-2.6-GCCcore-8.3.0.eb]: @edmondac This easyconfigs should be removed, `GSL` is performance sensitive so we're using `GCC` and `iccifort` as toolchain there (see #9504)

Comment (edmondac) [reply easybuild/easyconfigs/g/GSL/GSL-2.6-GCCcore-8.3.0.eb]: Done

**Comment by edmondac** [comment]
I won't be able to upload test reports to this before tomorrow, we've got a maintenance window today

File: easybuild/easyconfigs/r/R/R-3.6.2-foss-2019b.eb [hunk 2]
```diff
@@ -0,0 +1,2305 @@
...
+        'checksums': ['466d428d25c066c9c96d892f24da930513d42b1bdf76d3b53628c3ba13c3e48a'],
+    }),
+    ('OceanView', '1.0.4', {
+        'checksums': ['e67f6f376737e1e5cf22fdfe2769a8f674e90c886b0e43942e97d9a3e6924f1c'],
+    }),
+    ('ks', '1.11.6', {
+        'checksums': ['d8f1ccb99281eed092b8f950cfbf422330e88f2a0ce1ef6c9dbd7d8b9d648c41'],
+    }),
+    ('logcondens', '2.1.5', {
+        'checksums': ['72e
```
Comment (boegel) [inline easybuild/easyconfigs/r/R/R-3.6.2-foss-2019b.eb]: @edmondac This depends on `Biobase`, which is a Bioconductor package (so it belongs in that `R-bundle-Bioconductor` easyconfig instead)...

Comment (edmondac) [reply easybuild/easyconfigs/r/R/R-3.6.2-foss-2019b.eb]: How odd, it's been in the main R build since 2016a.

Are you sure? I can't see any mention of Bio here: https://cran.r-project.org/web/packages/logcondens/index.html

Comment (edmondac) [reply easybuild/easyconfigs/r/R/R-3.6.2-foss-2019b.eb]: Ah - "ERROR: dependency Biobase is not available for package multtest"

Comment (boegel) [reply easybuild/easyconfigs/r/R/R-3.6.2-foss-2019b.eb]: Yeah, this was supposed to go where `multtest` is, sorry about the confusion. :)

**Comment by boegel** [comment]
@edmondac `mutoss` will need to be removed too, since it depends on `multtest`

```
ERROR: dependency multtest is not available for package mutoss
```

**Comment by edmondac** [comment]
OK - I'm uploading test reports now - which will take a little while...

**Comment by edmondac** [comment]
I'm also just checking it'll build on POWER9...

**Comment by edmondac** [comment]
I'll put the POWER support changes into a new PR

**Comment by edmondac** [comment]
Test report by @bear-rsg
**FAILED**
Build succeeded for 7 out of 8 (8 easyconfigs in this PR)
ed-Q170M-C - Linux ubuntu 19.10, Intel(R) Core(TM) i5-6500 CPU @ 3.20GHz, Python 3.7.5
See https://gist.github.com/c2dcc4b740c2a83e8266140d28d00ea6 for a full test report.

**Comment by edmondac** [comment]
> Test report by @bear-rsg
> **FAILED**
> Build succeeded for 7 out of 8 (8 easyconfigs in this PR)
> ed-Q170M-C - Linux ubuntu 19.10, Intel(R) Core(TM) i5-6500 CPU @ 3.20GHz, Python 3.7.5
> See https://gist.github.com/c2dcc4b740c2a83e8266140d28d00ea6 for a full test report.

That failed because the machine I had to use this morning doesn't have libcuda :-( Doh!

**Comment by edmondac** [comment]
Test report by @bear-rsg
**FAILED**
Build succeeded for 5 out of 8 (8 easyconfigs in this PR)
bear-pg0212u15b.bear.cluster - Linux centos linux 7.7.1908, Intel(R) Xeon(R) CPU E5-2640 v4 @ 2.40GHz, Python 2.7.5
See https://gist.github.com/13a54087b0089d2171ade6c7f75eabd8 for a full test report.

**Comment by edmondac** [comment]
Test report by @bear-rsg
**FAILED**
Build succeeded for 16 out of 17 (8 easyconfigs in this PR)
bear-pg0212u15b.bear.cluster - Linux centos linux 7.7.1908, Intel(R) Xeon(R) CPU E5-2640 v4 @ 2.40GHz, Python 2.7.5
See https://gist.github.com/45fe8f1c7a67f0751aae3c756cd83972 for a full test report.

**Comment by edmondac** [comment]
> Test report by @bear-rsg
> **FAILED**
> Build succeeded for 16 out of 17 (8 easyconfigs in this PR)
> bear-pg0212u15b.bear.cluster - Linux centos linux 7.7.1908, Intel(R) Xeon(R) CPU E5-2640 v4 @ 2.40GHz, Python 2.7.5
> See https://gist.github.com/45fe8f1c7a67f0751aae3c756cd83972 for a full test report.

I'm not having much luck! `download from 'http://s3.amazonaws.com/h2o-release/h2o/rel-yau/2/Rjar/h2o.jar' failed`

**Comment by edmondac** [comment]
Test report by @bear-rsg
**SUCCESS**
Build succeeded for 8 out of 8 (8 easyconfigs in this PR)
bear-pg0212u15b.bear.cluster - Linux centos linux 7.7.1908, Intel(R) Xeon(R) CPU E5-2640 v4 @ 2.40GHz, Python 2.7.5
See https://gist.github.com/bb5b08ede10cdf118532acf3995e3038 for a full test report.

**Comment by lexming** [comment]
Test report by @lexming
**FAILED**
Build succeeded for 13 out of 14 (8 easyconfigs in this PR)
login1.cerberus.os - Linux centos linux 7.6.1810, Intel(R) Xeon(R) Gold 6126 CPU @ 2.60GHz, Python 2.7.5
See https://gist.github.com/72ee814a1ea4258dce580b7eaf1d06c2 for a full test report.

**Comment by lexming** [comment]
Test report by @lexming
**FAILED**
Build succeeded for 12 out of 14 (8 easyconfigs in this PR)
node157.hydra.os - Linux centos linux 7.6.1810, Intel(R) Xeon(R) CPU E5-2680 v2 @ 2.80GHz, Python 2.7.5
See https://gist.github.com/d02091ab1a65e4be0a75dabdb9029715 for a full test report.

**Comment by lexming** [comment]
The checksum of `MASS_7.3-51.4.tar.gz` seems to have changed. I got one with SHA256
`844270a2541eaed420871dfb61d681aa67ee57126645fb6b144b436c25698eeb`

**Comment by boegel** [comment]
> The checksum of `MASS_7.3-51.4.tar.gz` seems to have changed. I got one with SHA256
> `844270a2541eaed420871dfb61d681aa67ee57126645fb6b144b436c25698eeb`

Confirmed, I'm getting the same for a fresh download...

As usual, the only difference is in the package description:

```
$ diff -ru MASS.old MASS.new
diff -ru MASS.old/DESCRIPTION MASS.new/DESCRIPTION
--- MASS.old/DESCRIPTION	2019-04-26 15:18:59.000000000 +0200
+++ MASS.new/DESCRIPTION	2019-03-31 09:55:05.000000000 +0200
@@ -33,4 +33,4 @@
   David Firth [ctb]
 Maintainer: Brian Ripley <ripley@stats.ox.ac.uk>
 Repository: CRAN
-Date/Publication: 2019-04-26 13:18:59 UTC
+Date/Publication: 2019-03-31 07:55:05 UTC
diff -ru MASS.old/MD5 MASS.new/MD5
--- MASS.old/MD5	2019-04-26 15:18:59.000000000 +0200
+++ MASS.new/MD5	2019-03-31 09:55:05.000000000 +0200
@@ -1,4 +1,4 @@
-a2dbb5e4e548578e5d6fce1c6683d531 *DESCRIPTION
+cd846d9af9038c65eb2b8bde3fa0952f *DESCRIPTION
 35aff05a505ecf7e81e0473767794ca9 *INDEX
 e0d1ab11f24f7e3ff569541429d65825 *LICENCE.note
 0ac7b30ad35a4c19ea69d76a6a366b02 *NAMESPACE
```

@edmondac Please add two acceptable checksums for `MASS`, like this:

```python
    ('MASS', '7.3-51.4', {
        'checksums': [('9911d546a8d29dc906b46cb53ef8aad76d23566f4fc3b52778a1201f8a9b2c74',
                       '844270a2541eaed420871dfb61d681aa67ee57126645fb6b144b436c25698eeb')],
    })

**Comment by boegel** [comment]
@edmondac This also needs a sync with `develop` after the merge of #9531...

**Comment by edmondac** [comment]
Test report en route...

**Comment by edmondac** [comment]
Test report by @bear-rsg
**FAILED**
Build succeeded for 6 out of 7 (7 easyconfigs in this PR)
bear-pg0211u03a.bear.cluster - Linux centos linux 7.7.1908, Intel(R) Xeon(R) Gold 6248 CPU @ 2.50GHz, Python 2.7.5
See https://gist.github.com/7f75bdddce3f4e4ab4c074dedd34fb35 for a full test report.

**Comment by edmondac** [comment]
> Test report by @bear-rsg
> **FAILED**
> Build succeeded for 6 out of 7 (7 easyconfigs in this PR)
> bear-pg0211u03a.bear.cluster - Linux centos linux 7.7.1908, Intel(R) Xeon(R) Gold 6248 CPU @ 2.50GHz, Python 2.7.5
> See https://gist.github.com/7f75bdddce3f4e4ab4c074dedd34fb35 for a full test report.

I didn't build it on a GPU machine, hence the CUDA missing deps :-(

**Comment by edmondac** [comment]
All our X86 GPU nodes are currently busy, so I can't run a test report on those right now...

**Comment by edmondac** [comment]
Test report by @bear-rsg
**FAILED**
Build succeeded for 5 out of 7 (7 easyconfigs in this PR)
bear-pg0212u15b.bear.cluster - Linux centos linux 7.7.1908, Intel(R) Xeon(R) CPU E5-2640 v4 @ 2.40GHz, Python 2.7.5
See https://gist.github.com/045b63902b028929e480bd640a85cfd3 for a full test report.

**Comment by edmondac** [comment]
Test report by @bear-rsg
**SUCCESS**
Build succeeded for 14 out of 14 (7 easyconfigs in this PR)
bear-pg0212u17a.bear.cluster - Linux centos linux 7.7.1908, Intel(R) Xeon(R) CPU E5-2640 v4 @ 2.40GHz, Python 2.7.5
See https://gist.github.com/7a03592b6db1e63cdf17d52159c9e291 for a full test report.

**Comment by zao** [comment]
Seems like the `boot` extension checksum has changed:
`== 2020-01-07 11:05:31,748 filetools.py:825 DEBUG Computed sha256 checksum for /hp/eb/sources/r/R/extensions/boot_1.3-23.tar.gz: 79236a5a770dc8bf5ce25d9aa303c5dc0574d94aa043fd00b8b4c8ccc877357f (correct checksum: 30c
89e19dd6490b943233e87dfe422bfef92cfbb7a7dfb5c17dfd9b2d63fa02f)`

**Comment by edmondac** [comment]
These checksum changes are really frustrating!

**Comment by edmondac** [comment]
```
tmp $ diff -ur boot.old/ boot
diff -ur boot.old/DESCRIPTION boot/DESCRIPTION
--- boot.old/DESCRIPTION        2019-07-05 14:41:45.000000000 +0100
+++ boot/DESCRIPTION    2019-07-05 14:41:12.000000000 +0100
@@ -24,4 +24,4 @@
 Author: Angelo Canty [aut],
   Brian Ripley [aut, trl, cre] (author of parallel support)
 Repository: CRAN
-Date/Publication: 2019-07-05 13:41:45 UTC
+Date/Publication: 2019-07-05 13:41:12 UTC
diff -ur boot.old/MD5 boot/MD5
--- boot.old/MD5        2019-07-05 14:41:45.000000000 +0100
+++ boot/MD5    2019-07-05 14:41:12.000000000 +0100
@@ -1,5 +1,5 @@
 69ad56735054de20c73f6cfa94a7bf1e *ChangeLog
-e1f55e7222db6038a9a36eb6b3b7fc15 *DESCRIPTION
+b1407dc339d81e7c7a66a946edfe5a2c *DESCRIPTION
 e6929f73b03924d15f3af4a4c8549a45 *INDEX
 52cc784fd6ef9af32d9586726a53bb89 *NAMESPACE
 319ea125989deab41f19cb6c3071aecb *R/bootfuns.q

```

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 7 out of 7 (7 easyconfigs in this PR)
node3308.joltik.os - Linux centos linux 7.7.1908, Intel(R) Xeon(R) Gold 6242 CPU @ 2.80GHz, Python 3.6.8
See https://gist.github.com/e1d8d8e479b46a8a0bf8f878e2ccfbe5 for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**FAILED**
Build succeeded for 6 out of 7 (7 easyconfigs in this PR)
node3115.skitty.os - Linux centos linux 7.7.1908, Intel(R) Xeon(R) Gold 6140 CPU @ 2.30GHz, Python 3.6.8
See https://gist.github.com/2728822b7fc69da80bfd003837723e50 for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**FAILED**
Build succeeded for 6 out of 7 (7 easyconfigs in this PR)
node2406.golett.os - Linux centos linux 7.7.1908, Intel(R) Xeon(R) CPU E5-2680 v3 @ 2.50GHz, Python 2.7.5
See https://gist.github.com/f25f28b8ba433164540df139d6688cbb for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**FAILED**
Build succeeded for 6 out of 7 (7 easyconfigs in this PR)
generoso-1 - Linux centos linux 7.6.1810, Intel(R) Xeon(R) CPU E5-2660 v4 @ 2.00GHz, Python 3.6.8
See https://gist.github.com/ce098fcf0640db9c5f7deda069660809 for a full test report.

**Comment by zao** [comment]
`gpuR` fails to build on my machine (i7 8700K, GTX 1080):
(tail of log) https://gist.github.com/zao/13723ba5745dbdc7e5585af26c966d06

Seems like it bails out on not finding any OpenCL devices, which apparently is a fatal error?
Are we missing some dependency here, operating system or not?

**Comment by boegel** [comment]
@zao Relevant output from our GPU box with V100:

```
** testing if installed package can be loaded from temporary location
Number of platforms: 1                                                                                                                                                                                      - platform: NVIDIA Corporation: OpenCL 1.2 CUDA 10.1.236
  - context device index: 0
    - Tesla V100-SXM2-32GB
checked all devices                                                                                                                                                                                         completed initialization
```

Why is it finding 2 platforms in your case? Could that be the issue?

**Comment by zao** [comment]
The machine has the system Mesa ICD installed, assumedly as a dependency of something else OS-wise.
The NV ICD is part of the driver package and is also present.

`clinfo` output seems a bit patchy toward the end: https://gist.github.com/zao/90fcab18037cb4d6c5119b84d0b6ecd7

**Comment by edmondac** [comment]
```
tmp $ diff -ur rda.old/ rda/
diff -ur rda.old/DESCRIPTION rda/DESCRIPTION
--- rda.old/DESCRIPTION 2018-07-21 18:12:01.000000000 +0100
+++ rda/DESCRIPTION     2020-01-01 15:58:30.000000000 +0000
@@ -17,3 +17,5 @@
 X-CRAN-Original-Maintainer: Rob Tibshirani <tibs@stanford.edu>
 X-CRAN-Comment: Orphaned and corrected on 2018-07-21 as check problems
         were not corrected despite reminders.
+ .
+ BioC:MLInterfaces depends on this
diff -ur rda.old/MD5 rda/MD5
--- rda.old/MD5 2018-07-21 18:12:01.000000000 +0100
+++ rda/MD5     2020-01-01 15:58:30.000000000 +0000
@@ -1,4 +1,4 @@
-25d1293d678d5fbdf2bb20bf6fc64280 *DESCRIPTION
+2a1d3c1ed3cac2196abc5a33b27b7639 *DESCRIPTION
 61ebf1914538b6254a39f161f286069f *INDEX
 63d89739955fd7c7e2463bb9b8ad087c *NAMESPACE
 bb8590ce1b9dc15b42c525b8be97a5f4 *R/Drda1.R
```

**Comment by zao** [comment]
Test report by @zao
**FAILED**
Build succeeded for 1 out of 3 (7 easyconfigs in this PR)
freja - Linux ubuntu 18.04, Intel(R) Core(TM) i7-8700K CPU @ 3.70GHz, Python 2.7.17
See https://gist.github.com/a36cbdf199b4929304483166f116bad0 for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 6 out of 6 (1 easyconfigs in this PR)
node3109.skitty.os - Linux centos linux 7.7.1908, Intel(R) Xeon(R) Gold 6140 CPU @ 2.30GHz, Python 3.6.8
See https://gist.github.com/d3f92c79fb96c58985e53b0da4c7ef0e for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 6 out of 6 (1 easyconfigs in this PR)
node2461.golett.os - Linux centos linux 7.7.1908, Intel(R) Xeon(R) CPU E5-2680 v3 @ 2.50GHz, Python 2.7.5
See https://gist.github.com/1eeff3891c6e870aabc1fe393ac42222 for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
node3308.joltik.os - Linux centos linux 7.7.1908, Intel(R) Xeon(R) Gold 6242 CPU @ 2.80GHz, Python 3.6.8
See https://gist.github.com/6e8257e04be2a6ee1ff13097b39a8cd3 for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 6 out of 6 (1 easyconfigs in this PR)
generoso-1 - Linux centos linux 7.6.1810, Intel(R) Xeon(R) CPU E5-2660 v4 @ 2.00GHz, Python 3.6.8
See https://gist.github.com/68e5a39179ae10a346f29aa5b96831da for a full test report.

**Comment by edmondac** [comment]
Test report by @bear-rsg
**FAILED**
Build succeeded for 6 out of 7 (7 easyconfigs in this PR)
bear-pg0212u15b.bear.cluster - Linux centos linux 7.7.1908, Intel(R) Xeon(R) CPU E5-2640 v4 @ 2.40GHz, Python 2.7.5
See https://gist.github.com/4751a30b69ccd882b166179e76efbcb1 for a full test report.

**Comment by lexming** [comment]
Test report by @lexming
**FAILED**
Build succeeded for 11 out of 13 (7 easyconfigs in this PR)
node157.hydra.os - Linux centos linux 7.7.1908, Intel(R) Xeon(R) CPU E5-2680 v2 @ 2.80GHz, Python 2.7.5
See https://gist.github.com/b61318100fea0120da4af4440aedcbf2 for a full test report.

**Comment by lexming** [comment]
Ok,`R/3.6.2-foss-2019b` built successfully.
My problems are related to incompatible `CUDA` modules
* `fosscuda/2019b` depends on `CUDA/10.1.243`
* `Clang/9.0.1-GCC-8.3.0-CUDA-10.1.243` depends on `CUDA/10.1.243-GCC-8.3.0`

**Comment by branfosj** [comment]
@lexming `fosscuda/2019b` (and `gcccuda/2019b` and `gompic/2019b`) have been changed. See https://github.com/easybuilders/easybuild-easyconfigs/pull/9178

**Comment by lexming** [comment]
@branfosj thanks, rebuilding them and retesting :+1:

**Comment by edmondac** [comment]
Test report by @bear-rsg
**SUCCESS**
Build succeeded for 7 out of 7 (7 easyconfigs in this PR)
bear-pg0212u15b.bear.cluster - Linux centos linux 7.7.1908, Intel(R) Xeon(R) CPU E5-2640 v4 @ 2.40GHz, Python 2.7.5
See https://gist.github.com/fdb4f52ec9a5e5b27302a78c4a23ad99 for a full test report.

**Comment by lexming** [comment]
Test report by @lexming
**SUCCESS**
Build succeeded for 8 out of 8 (7 easyconfigs in this PR)
node157.hydra.os - Linux centos linux 7.7.1908, Intel(R) Xeon(R) CPU E5-2680 v2 @ 2.80GHz, Python 2.7.5
See https://gist.github.com/8bf9c9f560270d70951cd076f4990176 for a full test report.

File: easybuild/easyconfigs/r/R/R-3.6.2-fosscuda-2019b.eb [hunk 3]
```diff
@@ -0,0 +1,2356 @@
...
+    ('plot3D', '1.1.1', {
+        'checksums': ['f6fe4a001387132626fc553ed1d5720d448b8064eb5a6917458a798e1d381632'],
+    }),
+    ('plot3Drgl', '1.0.1', {
+        'checksums': ['466d428d25c066c9c96d892f24da930513d42b1bdf76d3b53628c3ba13c3e48a'],
+    }),
+    ('OceanView', '1.0.4', {
+        'checksums': ['e67f6f376737e1e5cf22fdfe2769a8f674e90c886b0e43942e97d9a3e6924f1c'],
+    }),
+    ('ks',
```
Comment (smoors) [inline easybuild/easyconfigs/r/R/R-3.6.2-fosscuda-2019b.eb]: @edmondac why is this needed?

File: easybuild/easyconfigs/r/R/R-3.6.2-foss-2019b.eb [hunk 4]
```diff
@@ -0,0 +1,2299 @@
...
+    }),
+    ('plot3D', '1.1.1', {
+        'checksums': ['f6fe4a001387132626fc553ed1d5720d448b8064eb5a6917458a798e1d381632'],
+    }),
+    ('plot3Drgl', '1.0.1', {
+        'checksums': ['466d428d25c066c9c96d892f24da930513d42b1bdf76d3b53628c3ba13c3e48a'],
+    }),
+    ('OceanView', '1.0.4', {
+        'checksums': ['e67f6f376737e1e5cf22fdfe2769a8f674e90c886b0e43942e97d9a3e6924f1c'],
+
```
[the remark this answers is not in the document]
Comment (edmondac) [reply easybuild/easyconfigs/r/R/R-3.6.2-foss-2019b.eb]: OceanView's been in R easyconfigs for several versions - so what's the question?

File: easybuild/easyconfigs/r/R/R-3.6.2-fosscuda-2019b.eb (hunk 3 above)
Comment (edmondac) [reply easybuild/easyconfigs/r/R/R-3.6.2-fosscuda-2019b.eb]: Ditto, ks has been there for ages...?

File: easybuild/easyconfigs/r/R/R-3.6.2-foss-2019b.eb (hunk 4 above)
[the remark this answers is not in the document]
Comment (edmondac) [reply easybuild/easyconfigs/r/R/R-3.6.2-foss-2019b.eb]: Hmm. How odd. In the diff this is next to 'tergm'.

[the remark this answers is not in the document]
Comment (edmondac) [reply easybuild/easyconfigs/r/R/R-3.6.2-foss-2019b.eb]: That's an artefact created by inject-checksums, I think, where the version of the extension is the same as the R version. I was probably looking at 3.6.1 before moving to 3.6.2...

**Comment by boegel** [comment]
Test report by @boegel
**FAILED**
Build succeeded for 0 out of 1 (1 easyconfigs in this PR)
node3301.joltik.os - Linux centos linux 7.7.1908, Intel(R) Xeon(R) Gold 6242 CPU @ 2.80GHz, Python 3.6.8
See https://gist.github.com/49846347b9c0f21b7128d8366bd452df for a full test report.

**Comment by edmondac** [comment]
That's very sad - and I should have tested it myself. I'll dig out easy_update again...

**Comment by edmondac** [comment]
I'll upload another test report

**Comment by boegel** [comment]
Test report by @boegel
**FAILED**
Build succeeded for 0 out of 1 (1 easyconfigs in this PR)
node2420.golett.os - Linux centos linux 7.7.1908, Intel(R) Xeon(R) CPU E5-2680 v3 @ 2.50GHz, Python 2.7.5
See https://gist.github.com/c72337918a87fa56d8363e26e3012c0e for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**FAILED**
Build succeeded for 0 out of 1 (1 easyconfigs in this PR)
node3301.joltik.os - Linux centos linux 7.7.1908, Intel(R) Xeon(R) Gold 6242 CPU @ 2.80GHz, Python 3.6.8
See https://gist.github.com/a8fb50bb9d023e7c8488ca75d26f576f for a full test report.

**Comment by branfosj** [comment]
The build failure was: 
> ERROR: dependency Biobase is not available for package multtest

We've gone round in a circle here. https://github.com/easybuilders/easybuild-easyconfigs/pull/9519/commits/ae65dbeed85fb285e03727abd7cd2273e285a0fc and https://github.com/easybuilders/easybuild-easyconfigs/pull/9519/commits/1510914ff78dc64589f710233122be2bf37aac7c removed `multtest`, `mutoss`, and `metap` due to the dependency on `Biobase`.

**Comment by edmondac** [comment]
Test report by @bear-rsg
**FAILED**
Build succeeded for 5 out of 7 (7 easyconfigs in this PR)
bear-pg0212u15a.bear.cluster - Linux centos linux 7.7.1908, Intel(R) Xeon(R) CPU E5-2640 v4 @ 2.40GHz, Python 2.7.5
See https://gist.github.com/9547881aee4a248c571b307f32cd0edc for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**FAILED**
Build succeeded for 0 out of 1 (1 easyconfigs in this PR)
generoso - Linux centos linux 7.6.1810, Intel(R) Xeon(R) CPU E5-2660 v4 @ 2.00GHz, Python 3.6.8
See https://gist.github.com/aa59d6ac56c0ca2404441da7ac5be202 for a full test report.

**Comment by edmondac** [comment]
@smoors I've had to remove metap (and multtest and mutoss) again as they depend on Biobase. See https://github.com/easybuilders/easybuild-easyconfigs/pull/9519#discussion_r358941228

New test report coming soon

**Comment by edmondac** [comment]
Christmas has wiped my memory - sorry for the confusion!

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
node3301.joltik.os - Linux centos linux 7.7.1908, Intel(R) Xeon(R) Gold 6242 CPU @ 2.80GHz, Python 3.6.8
See https://gist.github.com/ac205f298ddb99929f31ab60d612b1ee for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
generoso - Linux centos linux 7.6.1810, Intel(R) Xeon(R) CPU E5-2660 v4 @ 2.00GHz, Python 3.6.8
See https://gist.github.com/1e125da7eb50c77663301ec4d9ceb6ff for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
node3102.skitty.os - Linux centos linux 7.7.1908, Intel(R) Xeon(R) Gold 6140 CPU @ 2.30GHz, Python 3.6.8
See https://gist.github.com/333f53eddaf62ce4b0eb6074666c99a5 for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
node2437.golett.os - Linux centos linux 7.7.1908, Intel(R) Xeon(R) CPU E5-2680 v3 @ 2.50GHz, Python 2.7.5
See https://gist.github.com/a2673527bda1ed7053faa4b887618f4d for a full test report.

**Comment by boegel** [comment]
> @smoors I've had to remove metap (and multtest and mutoss) again as they depend on Biobase. See [#9519 (comment)](https://github.com/easybuilders/easybuild-easyconfigs/pull/9519#discussion_r358941228)

We should add `metap` to `R-bundle-Bioconductor` instead.

**Comment by edmondac** [comment]
Test report by @bear-rsg
**SUCCESS**
Build succeeded for 7 out of 7 (7 easyconfigs in this PR)
bear-pg0212u15b.bear.cluster - Linux centos linux 7.7.1908, Intel(R) Xeon(R) CPU E5-2640 v4 @ 2.40GHz, Python 2.7.5
See https://gist.github.com/1448fc4641e7c56bf9ce47fa60cc542b for a full test report.

**Comment by edmondac** [comment]
> > @smoors I've had to remove metap (and multtest and mutoss) again as they depend on Biobase. See [#9519 (comment)](https://github.com/easybuilders/easybuild-easyconfigs/pull/9519#discussion_r358941228)
> 
> We should add `metap` to `R-bundle-Bioconductor` instead.

Will do

**Comment by boegel** [comment]
Going in, thanks @edmondac!
